embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testframework/actions/EmbracePostSetupInterface.kt

```diff
@@ -0,0 +1,13 @@
+package io.embrace.android.embracesdk.testframework.actions
+
+import io.embrace.android.embracesdk.Embrace
+import io.embrace.android.embracesdk.fakes.FakeClock
+
+internal class EmbracePostSetupInterface(
```

Comment:
Would suggest renaming this `EmbracePreSdkInitInterface` or something similar, initially I was a bit confused about why this needed to be a separate thing to `testCaseAction`

Reply:
Sounds good